centos-7.9/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM centos:7.9.2009
 RUN useradd -ms /bin/bash easybuild
-RUN yum -y install epel-release && yum -y install python python2-pip Lmod
+RUN yum -y install epel-release && yum -y install python3 python3-pip Lmod
```

Comment:
I would prefer keeping `python` in the CentOS 7 container, otherwise we don't have an easy way of testing EB with Python 2.7

We can install `archspec` with `pip install 'archspec<0.2.0'`

Reply:
We could also add `centos-7.9-python3/Dockerfile`, that would definitely make sense as an additional test environment

Reply:
Okay, that is a good reason to stick to Python 2. :+1: 

I think we only need a newer but still with Python2 compatible version of `pip` to install `archspec`. I changed this in https://github.com/easybuilders/easybuild-containers/pull/18/commits/eacda5c87bf0bcd01bdf22c59f1b1e1c614dca49

Reply:
Hmm, but `pip==20.3.4` will actually install `archspec==0.1.4`. So `archspec<0.2.0` would have been sufficient.

Reply:
This is fine too, I guess pip 20.3.4 is smart enough to pick the archspec version that's still compatible with Python 2 automagically...